codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -241,17 +246,18 @@ public void receiveFlowControlledFrame(Http2Stream stream, ByteBuf data, int pad
         assert ctx != null && ctx.executor().inEventLoop();
         int dataLength = data.readableBytes() + padding;
 
-        // Apply the connection-level flow control
-        FlowState connectionState = connectionState();
-        connectionState.receiveFlowControlledFrame(dataLength);
-
-        if (stream != null && !isClosed(stream)) {
```

Comment:
It seems like the code change is roughly equivalent now ... can this be reverted?